packages/gatsby/src/redux/run-fast-filters.ts

```diff
@@ -1,45 +1,60 @@
-// @flow
-
-const { prepareRegex } = require(`../utils/prepare-regex`)
+import { IGatsbyNode } from "./types"
+import { GatsbyGraphQLType } from "../../index"
```

Comment:
```suggestion
import { GatsbyGraphQLType } from "../.."
```
Don't explicitly import from index